handy-pandas/djangazon
# Am user complete order

## Description
1. Name of Branch:
```am-user-complete-order```

2. Tickets related to PR (with links):
[User Can Complete an Order](https://github.com/handy-pandas/djangazon/issues/2)

3. Feature/Fix Description:
- Added views for order, payment type, confirmation of order
- Added templates for order, payment type, confirmation of order

4. What architectual changes made:
- Added files corresponding to views, templates of order, payment, confirmation

5. Any dependency changes:
No

## Testing
*To Test:
- Please ```python manage.py deletedb```
- ```python manage.py builddb```
- ```python manage.py runserver```

- confirm operation of shopping cart view
    - Does not allow continue if no products show
        - To demonstrate below please go to line 24 in view_order.py and replace that line with
        - ```order = Order.objects.get(user__id=1, payment=None)```
    - Does not allow continue if no payments show to select from.

1. New Unit Tests:
No

2. All Tests Pass:
pre-existing tests yes

## Documentation
1. Fully completed Documentation with signature:
Yes sir

2. Readme file updated:
no

## Deployment
1. Any new migrations to run:
yes, please use
- ```python manage.py deletedb```
- ```python manage.py builddb```
